content/docs/start/data-and-model-versioning.md

```diff
@@ -59,7 +59,7 @@ $ git commit -m "Add raw data"
 
 The data, meanwhile, is listed in `.gitignore`.
 
-<details>
+<details id="expand-dvc-add">
```

Comment:
As Roger commented, it would be better to take `id` to override the anchor link.
The anchor for this `id` will be `#expand-dvc-add`

> I'm curious about this. Is that necessary? For example if we have repeated titles e.g. ## Something and ### Something there's no build error even when technically it's a violation of HTML rules. Maybe a warning is enough?

Since, the whole idea is to have an anchor link unique, and robust(i.e doesn't change on moving content block to another section). To make sure of that we need to throw an error, instead of letting it pass.

![screenshot-localhost_8000-2022 04 02-07_13_28](https://user-images.githubusercontent.com/20840228/161360500-cddacb58-176f-4fc8-92cc-abe57781df58.png)

I think this will be descriptive enough for us to fix that.

> That said we do need to update all `<details>` blocks regardless to move first-child text headers into title or id.

For title as props, we have a separate discussion over #3388. 
So, 
- the title will still be the first child
- the title will be used to generate the unique anchor link
- if there are two or more same titles on the page, there will be an error that will need manual intervention to make that unique
- `id` as props will override the use of the title to generate a unique anchor link
- No, we don't need to update all the `<details>` blocks but only the ones that have duplicate titles
- Also, setting `id` can also be helpful in the context where the title is long so can pass shorter `id` which will override the link

Reply:
> would be better to take id to override the anchor link

Agree, sounds like the easiest implementation and usage.

> To make sure of that we need to throw an error
this will be descriptive enough for us to fix that
I still think a warning is enough but not a strong opinion (up to you).

Reply:
> would be better to take id to override the anchor link

Agree, sounds like the easiest implementation and usage.

> To make sure of that we need to throw an error
this will be descriptive enough for us to fix that

I still think a warning is enough but not a strong opinion. Seems a bit out of scope for this PR to me but again: up to you.